File: fixtures/green/csharp/CyclomaticComplexity.cs
using System;
using System.Collections.Generic;

namespace Demo;

public class RouteScore
{
    public int Score(string kind, int retries, bool urgent)
    {
        var scores = new Dictionary<string, int>
        {
            ["create"] = 1,
            ["update"] = 1,
            ["delete"] = 1,
            ["manual"] = 1,
            ["batch"] = 1,
            ["sync"] = 1,
        };
        var score = scores.GetValueOrDefault(kind, 0) + Math.Min(Math.Max(retries, 0), 3);
        return urgent ? score + 1 : score;
    }
}

File: fixtures/green/python/cyclomatic_complexity.py
def route_score(kind: str, retries: int, urgent: bool) -> int:
    score = {"create": 1, "update": 1, "delete": 1, "manual": 1, "batch": 1, "sync": 1}.get(kind, 0)
    score += min(max(retries, 0), 3)
    return score + (1 if urgent else 0)
